crates/deno_task_shell/src/shell/commands/args.rs

```diff
@@ -76,9 +88,9 @@ mod test {
     assert_eq!(
       args,
       vec![
-        ArgKind::ShortFlag('f'),
-        ArgKind::ShortFlag('a'),
-        ArgKind::ShortFlag('b'),
+        ArgKind::MinusShortFlag('f'),
```

Comment:
I am wondering if we shoudl keep `ShortFlag` as is, and just add `PlusShortFlag`.
Also we could consider using `Dash` instead of `Minus`. But I think I would keep `ShortFlag` and add `PlusFlag` or `PlusShortFlag`.